packages/backend/src/database/migrations/20230327070336_add_developer_role.ts

```diff
@@ -0,0 +1,23 @@
+import { Knex } from 'knex';
+
+const organizationMembershipRolesTableName = 'organization_membership_roles';
+const projectMembershipRolesTableName = 'project_membership_roles';
+const developerRole = 'developer';
+
+export async function up(knex: Knex): Promise<void> {
```

Comment:
This would remove SQL runner access for all existing editors. Do we want to move editors to be developers?

Reply:
Added to migration

Reply:
@rephus we might need to split this over two migrations after the lesson with permissions updates

Reply:
done